### PR TITLE
feat: add 3.0.1

### PR DESCRIPTION
**Pull request by donstephan**
- support for 3.0.1 running with node 20.15.1
- removed the need for npm-check-updates installed globalled in favor of running with npx

### CODE REVIEW

**Comment by donstephan** [comment]
@GeoffreyBooth could you please merge?

**Comment by GeoffreyBooth** [comment]
What about version 3.0.0?

File: update.sh [hunk 1]
```diff
@@ -126,7 +119,7 @@ cd example/app
 
 meteor update --release "${new_meteor_version}"
 
-npm-check-updates --configFilePath /dev/null --upgrade
+npx npm-check-updates --configFilePath /dev/null --upgrade
```
Comment (GeoffreyBooth) [inline update.sh:122]: Can the use of `npx` be split out into its own PR? The version bump PRs should be essentially fully automatic.

File: test.sh [hunk 2]
```diff
@@ -39,7 +39,16 @@ for version in "${meteor_versions[@]}"; do
 
 	cp "${dockerfile}" test.dockerfile
 	do_sed "s|FROM geoffreybooth/meteor-base:.*|FROM geoffreybooth/meteor-base:${version}|" test.dockerfile
-	do_sed "s|FROM node:.*|FROM node:${node_version}-alpine|" test.dockerfile
+
+	# if we're using node version 14.21.4
+	# we need to get the meteor/node image
+	# else we use the official node image
```
Comment (GeoffreyBooth) [inline test.sh:45]:
```suggestion
	# For Node 14.21.4 we need to get the `meteor/node` image; otherwise we can use the official Node image
```

Comment (GeoffreyBooth) [reply test.sh:45]: Images were using the official Node image for 14.21.4 before this PR; why does this need to change? This also feels like it should be a separate PR, as it’s unrelated to adding Meteor 3.0.1.

File: update.sh (hunk 1 above)
Comment (donstephan) [reply update.sh:122]: Yes I'll split this up.

File: test.sh (hunk 2 above)
Comment (donstephan) [reply test.sh:45]: Tests were failing due to it trying to reference `node:14.21,4-alpine3.17` which only exists in the `meteor/node` registry. I'm really not sure how this logic was working before I made this change.

This sed statement would have no effect on this line:
```
export node_version=14.21.3
echo "FROM meteor/node:14.21.4-alpine3.17" | sed "s|FROM node:.*|FROM node:${node_version}-alpine|"
```

So I suspect all tests were running using `meteor/14.21.4-alpine3.17` but there's no output in old tests to confirm this. The test output changed in #144 is only informative and not alter that sed logic.

Here's output from the PR before I made this change: https://github.com/disney/meteor-base/actions/runs/9994011488/job/27622819639#step:4:30

Do you want this in a separate PR? This is a blocker for tests to pass.

**Comment by donstephan** [comment]
There doesn't seem to be a `3.0.0`. Forums reference `3.0.1` in the announcement. There some more information related to this in announcement for the v3 release: https://forums.meteor.com/t/its-official-meteor-3-0-official-release-is-out/61860

File: test.sh (hunk 2 above)
Comment (GeoffreyBooth) [reply test.sh:45]: > So I suspect all tests were running using `meteor/14.21.4-alpine3.17` but there’s no output in old tests to confirm this.

There’s GitHub Actions output for every PR that’s been merged in. Here’s the last one: https://github.com/disney/meteor-base/actions/runs/9227042103/job/25388152875

Comment (donstephan) [reply test.sh:45]: I'm going to close this and split it into separate PRs so this is more clear.
